db/migrate/20210706215522_create_reporting_matviews_with_conventional_names.rb

```diff
@@ -0,0 +1,13 @@
+class CreateReportingMatviewsWithConventionalNames < ActiveRecord::Migration[5.2]
+  def change
+    execute "SET LOCAL TIME ZONE '#{Rails.application.config.country[:time_zone]}'"
+
+    create_view :reporting_patient_blood_pressures, materialized: true
+    create_view :reporting_patient_visits, materialized: true
+    create_view :reporting_patient_states, materialized: true
+
+    drop_view :reporting_patient_states_per_month, revert_to_version: 2, materialized: true
```

Comment:
One slightly annoying thing is that we can't delete these old SQL files from db/views, because it would break past migrations.

Once we roll up migrations, we could remove the old files.